modules/core/ante/ante_test.go

```diff
@@ -341,6 +361,20 @@ func (suite *AnteTestSuite) TestAnteDecorator() {
 			},
 			true,
 		},
+		{
+			"success on app callback error, app callbacks are skipped for performance",
```

Comment:
deleting this test. No mock module exists so I would have to add it. The test was extra defensive so that feels unnecessary